concent_api/verifier/tasks.py

```diff
@@ -105,17 +106,21 @@ def blender_verification_order(
 def verification_result(
     self,
     subtask_id:     str,
-    result:         VerificationResult,
+    result:         str,
     error_message:  Optional[str] = None,
-    error_code:     Optional[ErrorCode] = None,
+    error_code:     Optional[str] = None,
 ):
     logger.info(f'verification_result_task starts with: SUBTASK_ID {subtask_id} -- RESULT {result}')
 
     assert isinstance(subtask_id, str)
-    assert isinstance(result, VerificationResult)
+    assert isinstance(result, str)
+    assert result in VerificationResult.__members__.keys()
     assert isinstance(error_message, (str, type(None)))
-    assert isinstance(error_code, (ErrorCode, type(None)))
-    assert all([error_message, error_code]) if result == VerificationResult.ERROR else True
+    assert isinstance(error_code, (str, type(None)))
+
+    result_enum = VerificationResult[result]
+
+    assert all([error_message, error_code]) if result_enum == VerificationResult.ERROR else True
```

Comment:
`[error_message, error_code]` is not a boolean expression. Since it's a non-empty array, it's always true. What is it supposed to check?

Reply:
Sorry, I was wrong. It's fine. It does expect a list. I'm too used to the `all(condition for x in collection)` form :)